content/knowledge/scalinghistory.md

```diff
@@ -0,0 +1,135 @@
+   Staking is like putting some native cryptocurrency as a collateral.
+   The higher the stake and time in the game, the higher the chances of getting to write a block on behalf of the whole ecosystem and in return receive some cryptocurrency.
+   This can be distributed among the participating nodes in case of a node pool where there are multiple parties participating by delegating their tokens.
+   Two keywords are different here compared to PoW: (1) Computers that participate in PoS consensus are called Validators, (2) Once a block has been created and accepted by the network, it is said to be forged (not mined).
+   Then the validator gets some reward.
+   If a validator tries to be malicious then it gets penalised losing some of its staked cryptocurrency.
+   PoS is much faster than PoW as there is no need to solve complex cryptographic puzzles to get to be the block writer and the process is much more streamlined.
+   Also no unnecessary wastage of energy means PoS is more environment friendly.
+   The security of the network is ensured by the fact that if an attacker were to attack the system, it needs to have 51% of all the value staked.
+   As opposed to 51% of all computing power in case of a PoW system.
+   Sustainability, security are achieved in a PoS model like the one in ethereum2.0 which is on its way to be released.
+   PoS ensures more throughput compared to PoW model.
+   However there is a communication overhead which hinders scalability (TODO: Explain more).
+   Also the higher the adaptation, the higher the overall value represented by the overall native cryptocurrency and therefore higher the value that needs to be locked up as staking by the nodes.
+   This means the native currency's full potential isn't utilised as once they are locked, they can no longer be used for any other purposes unless they are unlocked after the locking period expires.
+   Network’s safety is proportional to the value of stake committed.
+
+## 3.1 Sharding and layer 2 solutions
+
+   For solving this scaling issue, 'sharding' is introduced(e.g. In case of [Ethereum2.0](https://www.youtube.com/watch?v=ctzGr58_jeI&t=657s&ab_channel=Finematics).
+   Sharding is a way of spreading the computing and storage workload from a blockchain network, so that each node no longer has to process the entire network's transactional load.
+   Each node only needs to maintain info corresponding to its specific shard or partition.
+   For example, addresses starting with “0x00” will be part of Shard1 and addresses starting with “0x01” will be part of Shard2 etc.
+   There is a mechanism in place for inter-shard communication so that the whole blockchain is still viewable from any node.
+   All the shards will be connected to the “Beacon chain” which will have all the history with it and acts as an orchestrator of the whole network.
+   The consensus between all the shards will be maintained by the Beacon chain.
```

Comment:
I would prefer to use **a** "beacon chain" here, because it's not the only way to shard a network.